mips/exc.S

```diff
@@ -94,6 +94,67 @@ user_exc_enter:
         addi    t0, 1
         sw      t0, TD_IDNEST(s0)
 
+
+
+        # Save FPU registers
+        # Skip if process is not using FPU
+        LOAD_PCPU(t0)
```

Comment:
You don't need to load `curthread` in here - it's already stored in `s0` register.